API.md

```diff
@@ -126,3 +126,53 @@ bool provision(const byte appEui[8], const appKey[16]);
 
 - `const byte appEui[8]`: Application Identifier for the device.
 - `const byte appKey[16]`: Application Key assigned to the device.
+
+## Class: TheThingsMessage
+
+```c
+#include <TheThingsNetwork.h>
+
+TheThingsMessage msg(Stream& modemStream);
+```
+
+- `Stream& debugStream`: Stream to write debug logs to (for The Things Node/Uno use `Serial` and data rate `9600`).
+
+## Method: encodeMessage
+Encode the message we want to send.
+
+```c
+void encodeMessage(api_Measurement measurement);
+```
+
+- `api_Measurement measurement`: Structure with all the variable we can send.
+
+## Method: sendMessage
```

Comment:
This requires `TheThingsNetwork` to include `TheThingsMessage`, which is beautiful but adds to the size. I'd rather have a `send()` in `TheThingsMessage` that calls `sendBytes()` of `TheThingsNetwork`